packages/vitest/src/node/cli/cli-config.ts

```diff
@@ -602,8 +602,26 @@ export const cliOptionsConfig: VitestCLIOptions = {
   },
   retry: {
     description:
-      'Retry the test specific number of times if it fails (default: `0`)',
-    argument: '<times>',
+      'Retry configuration. Can be a number or JSON string with advanced options (default: `0`)',
```

Comment:
It cannot be a JSON string, the value would be defined as `--retry.count=5 --retry.strategy=immediate`